src/services/findAllReferences.ts

```diff
@@ -1189,6 +1189,10 @@ namespace ts.FindAllReferences {
             }
             for (const indirectUser of indirectUsers) {
                 for (const node of getPossibleSymbolReferenceNodes(indirectUser, isDefaultExport ? "default" : exportName)) {
+                    if (getEffectiveModifierFlags(node.parent) & ModifierFlags.Export) {

```

Comment:
What exactly is this check doing?

Reply:
Ops. I pushed my debug code :). `indirect Users` includes the module that contains  `export` that will be changed. When `changeImports` tries to find import references, it finds `export` reference to itself., this is the wrong position, so QF throws an exception. I found that we can skip QF if the export name is equal to the found ref name. Do you think this is Ok? Or is it better to find a way to exclude such modules from `indirectUsers` in `importTracker`?